test/presubmit-tests.sh

```diff
@@ -23,36 +23,16 @@
 
 # Markdown linting failures don't show up properly in Gubernator resulting
 # in a net-negative contributor experience.
-
-export GO111MODULE=on
+export DISABLE_MD_LINK_CHECK=1
 export DISABLE_MD_LINTING=1
 
 source $(dirname $0)/../vendor/github.com/tektoncd/plumbing/scripts/presubmit-tests.sh
 
-unit_tests() {
- :
-}
-
-build_tests() {
-  header "Running operator-sdk build"
-  operator-sdk build gcr.io/tekton-nightly/tektoncd-operator
-}
-
-install_operator_sdk() {
-  local sdk_rel="v0.17.0"
-  curl -JL \
-    https://github.com/operator-framework/operator-sdk/releases/download/${sdk_rel}/operator-sdk-${sdk_rel}-x86_64-linux-gnu \
-    -o /usr/bin/operator-sdk
-  chmod +x /usr/bin/operator-sdk
-}
-
-extra_initialization() {
-  echo "Running as $(whoami) on $(hostname) under $(pwd) dir"
-
-  install_operator_sdk
-  echo ">> operator sdk version"
-  operator-sdk version
+function build_tests() {
```

Comment:
`build_tests` is for building the project. The least we could have here is a `go build` at least.